android/engine/src/main/java/org/smartregister/fhircore/engine/ui/login/LoginActivity.kt

```diff
@@ -54,6 +64,16 @@ class LoginActivity :
       configureViews(configurationRegistry.retrieveConfiguration(AppConfigClassification.LOGIN))
     }
 
+    // Check if Pin enabled and stored then move to Pin login
+    val isPinEnabled = loginViewModel.loginViewConfiguration.value?.enablePin ?: false
+    val stayUserNamePasswordLogin =
```

Comment:
for clarity, can we call this val forceLoginViaUsername

Reply:
updated